main.js

```diff
@@ -1,8 +1,8 @@
 // ------------------------------
 // Settings
 // ------------------------------
-const iconsBaseUrl = 'https://rawgit.com/atomiclabs/cryptocurrency-icons/master';
-const dataJson = 'https://rawgit.com/atomiclabs/cryptocurrency-icons/master/manifest.json';
+const iconsBaseUrl = 'https://cdn.rawgit.com/atomiclabs/cryptocurrency-icons/master';
+const dataJson = 'https://cdn.rawgit.com/atomiclabs/cryptocurrency-icons/master/manifest.json';
```

Comment:
We can't use the `master` branch if we use the CDN as it will then never update when we add new coins. We need to use a specific commit, but we also don't want to have to manually update the commit SHA every time we add a new icon, so I think the solution would be to fetch the latest commit SHA from the `cryptocurrency-icons` repo and use it here in the URL instead of the `master` branch.

Reply:
I replaced RawGit with jsDelivr since RawGit is shutting down. Also using the latest commit SHA to fetch them.